### PR TITLE
Fix Some Warnings from lgtm

### PR DESCRIPTION
**Pull request by naveen521kk**
## Motivation
I was looking at https://lgtm.com/projects/g/ManimCommunity/manim/alerts/?mode=list and found quite a few of these which I can be fixed easily. Maybe we should install that app?

## Explanation for Changes
Changes are pretty much trivial, if not it shouldn't be here.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/mobject/svg/opengl_text_mobject.py [hunk 1]
```diff
@@ -49,14 +49,10 @@ def construct(self):
 import hashlib
 import os
 import re
-import sys
 import typing
 from contextlib import contextmanager
 from pathlib import Path
-from typing import Dict
```
Comment (kolibril13) [inline manim/mobject/svg/opengl_text_mobject.py:56]: why this change? it is used in `t2c: Dict[str, str] = None`

Comment (naveen521kk) [reply manim/mobject/svg/opengl_text_mobject.py:56]: See https://lgtm.com/rules/1818040193/ 
https://lgtm.com/projects/g/ManimCommunity/manim/snapshot/506c4a87fbd35877151d92f507ef0a31f4b26d05/files/manim/mobject/svg/opengl_text_mobject.py?sort=name&dir=ASC&mode=heatmap#x50a2c203b0c6fa61:1

Comment (kolibril13) [reply manim/mobject/svg/opengl_text_mobject.py:56]: ok!

**Review by kolibril13 — APPROVED** [review]
